src/user/user.controller.ts

```
@@ -5,9 +5,13 @@ import {
   Get,
   Post,
   Query,
+  UsePipes,
```

ESLint

[error] 'UsePipes' is defined but never used.

(@typescript-eslint/no-unused-vars)